src/data/markdown/docs/01 guides/02 Using k6/07 Modules.md

```diff
@@ -384,3 +354,19 @@ Note that on Windows, you also need to make sure that your drive in question, sa
 has been marked for sharing in the Docker settings:
 
 ![Running k6 in docker on Windows](images/Modules/running-k6-in-docker-on-windows.png)
+
+## The JSLib repository
+
+[JSLib](https://jslib.k6.io/) is a set of libraries known to play nicely with k6, and available at https://jslib.k6.io/.
+
+- `k6-utils`
+- `jsonpath`
+- `form-urlencoded`
+- `papaparse`
+
+These libraries can either be downloaded and included with the test project or loaded directly using HTTP imports as shown [here](#remote-http-s-modules).
+
+## See also
```

Comment:
Would you mind making an MDX component out of this? Shouldn't be more than maybe 20 lines of code.

We're gonna want to use this in multiple places (we already have, but with difference headings and different ways of listing it) and would benefit from having the format constrained by a component.

Reply:
👍🏽